eng/common/templates/variables/docker-images.yml

```diff
@@ -1,5 +1,5 @@
 variables:
-  imageNames.imageBuilder: mcr.microsoft.com/dotnet-buildtools/image-builder:939194
+  imageNames.imageBuilder: mcr.microsoft.com/dotnet-buildtools/image-builder:20210108104530
```

Comment:
This needs to be updated with an official build with https://github.com/dotnet/docker-tools/pull/727.